addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/PathTraversalScanRule.java

```diff
@@ -45,7 +45,29 @@
 import org.zaproxy.zap.model.Vulnerabilities;
 import org.zaproxy.zap.model.Vulnerability;
 
-/** a scanner that looks for Path Traversal vulnerabilities */
+/**
+ * A scanner that looks for Path Traversal vulnerabilities (see <a
+ * href="https://owasp.org/www-community/attacks/Path_Traversal">OWASP Path Traversal</a>).
```

Comment:
I'm not sure the reference is worth adding. The commentary in the rule is really more for maintenance purposes not third parties. If someone wants to lookup info on path traversal it's a simple trip to their fav search engine.

Reply:
Since most scan rules don't have any references, we can go without here -> fixed